build(setup): remove install-yarn and document new yarn setup

**Describe the bug**

After running `npm run install-yarn` error is thrown:

```
vagrant@CactusDevVM:~/tmp/cacti$ npm run install-yarn

> install-yarn
> npm install --global yarn@3.6.0

npm ERR! code ETARGET
npm ERR! notarget No matching version found for yarn@3.6.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2023-07-25T12_26_15_488Z-debug-0.log
```

**Expected behavior**

I think setup of yarn 3.X changed and this script should be removed. We should document somewhere how to enable corepack or at least add link to https://yarnpkg.com/getting-started/install

Everything else seem to be working very well, thank you for upgrading yarn @petermetz!

--- a/.github/workflows/.dast-nuclei-cmd-api-server.yaml
+++ b/.github/workflows/.dast-nuclei-cmd-api-server.yaml
@@ -44,7 +44,7 @@ jobs:
 
       - run: nuclei --version
 
-      - run: yarn configure
+      - run: npm run configure
 
       - name: Create URLs file for Nuclei
         run: |

--- a/.github/workflows/ghpkg-all-kotlin-api-clients-publish.yaml
+++ b/.github/workflows/ghpkg-all-kotlin-api-clients-publish.yaml
@@ -47,7 +47,7 @@ jobs:
       with:
         node-version: v16.9.1
 
-    - run: yarn configure
+    - run: npm run configure
 
     - name: build-cactus-plugin-ledger-connector-corda-kotlin-client
       run: |

--- a/BUILD.md
+++ b/BUILD.md
@@ -100,7 +100,7 @@ The `npm run watch` script in action:
     nvm use 16.14.2
     ```
   * Yarn
-    * `npm run install-yarn` (from within the project directory)
+    * `npm run enable-corepack` (from within the project directory)
   * [Docker Engine](https://docs.docker.com/engine/install/ubuntu/). Make sure that Docker is working and running, for example, running ``docker ps -aq``
   * Docker Compose
   * OpenJDK (Corda support Java 8 JDK but do not currently support Java 9 or higher)

--- a/examples/supply-chain-app/README.md
+++ b/examples/supply-chain-app/README.md
@@ -54,16 +54,16 @@ DOCKER_BUILDKIT=1 docker build \
 
 On the terminal, issue the following commands:
 
-1. `npm run install-yarn`
-2. `yarn configure`
+1. `npm run enable-corepack`
+2. `npm run configure`
 3. `yarn start:example-supply-chain`
 
 ## Debugging the Example Application Locally
 
 On the terminal, issue the following commands (steps 1 to 6) and then perform the rest of the steps manually.
 
-1. `npm run install-yarn`
-2. `yarn configure`
+1. `npm run enable-corepack`
+2. `npm run configure`
 3. `yarn build:dev`
 4. `cd ./examples/supply-chain-app/`
 5. `yarn --no-lockfile`
@@ -77,8 +77,8 @@ On the terminal, issue the following commands (steps 1 to 6) and then perform th
 
 ## Live Reloading the GUI Application
 
-1. `npm run install-yarn`
-2. `yarn configure`
+1. `npm run enable-corepack`
+2. `npm run configure`
 3. `yarn build:dev`
 4. Locate the `.vscode/template.launch.json` file
 5. Within that file locate the entry named `"Example: Supply Chain App"`

--- a/package.json
+++ b/package.json
@@ -29,7 +29,6 @@
     "reset:yarn-lock": "yarn run init-registries && yarn install --update-checksums --force",
     "reset": "run-s reset:git reset:node-modules reset:yarn-lock configure",
     "configure": "npm run init-registries && yarn install && yarn build:dev:backend",
-    "install-yarn": "npm install --global yarn@3.6.0",
     "set-yarn-version": "yarn set version stable",
     "enable-corepack": "npm i -g corepack && corepack enable && corepack prepare yarn@3.6.0 --activate",
     "custom-checks": "TS_NODE_PROJECT=./tools/tsconfig.json node --trace-deprecation --experimental-modules --abort-on-uncaught-exception --loader ts-node/esm --experimental-specifier-resolution=node ./tools/custom-checks/run-custom-checks.ts",

